### DISCUSSION TITLE
Pivot-Table Style Table?

### DISCUSSION THREAD

**Discussion by Alex-Wenner-FHR** [h2oai/wave#1037] [Ideas] [answered]
A UI component that replicates the behavior of a Excel Pivot Table or grouped/aggregated Pandas data frame would be an awesome feature! There are a few good JavaScript implementations that I have seen out there. This would be helpful for various use cases. Currently I am using the standard table component and attempting to give that 'drill-down' behavior you get from a Pivot, using custom filters, table groupby, table filter and table search. Just a thought!

https://www.syncfusion.com/javascript-ui-controls/js-pivot-table
https://www.flexmonster.com/demos/pivot-table-js/

https://user-images.githubusercontent.com/78372056/135669347-5313e8a5-1a64-49ca-ac90-e77dfe1bcae5.MP4

https://user-images.githubusercontent.com/78372056/135669358-83134fd9-69f7-4a6b-8922-eb797efae46e.MP4

**Comment by mturoci** [comment, marked as answer]
Thanks @AlexWennerDT! That would be awesome feature.

Tracked in #1038.
